Calculator.py

```diff
@@ -26,4 +29,4 @@ def main():
     print(run_operation(number_1, number_2, operation))
 
 if __name__ == "__main__":
-	main()
\ No newline at end of file
+    main()
```

Comment:
Nice catch! We have to make sure we don't mix up tabs and spaces. This nicely fixes that